constants.py

```diff
@@ -4,7 +4,7 @@
 SCREEN_WIDTH = 600
 SCREEN_HEIGHT = 600
 SCREENRECT = pg.rect.Rect(0, 0, SCREEN_WIDTH, SCREEN_HEIGHT)
-fps = 30
+fps = 60
```

Comment:
```suggestion
fps = 30
```
Not sure if we need to increase fps, 30 should be fine. Is there a particular reason you increased fps?

Reply:
Oh I do not remember changing that, but if I did it was not intended. Thanks for the change!